packages/react-native/scripts/cocoapods/__tests__/test_utils/InstallerMock.rb

```diff
@@ -197,6 +197,10 @@ def initialize(name, build_settings = {}, is_debug: false)
     def debug?
       return @is_debug
     end
+
+    def type
+        @is_debug ? :debug : :release
```

Comment:
@cipolleschi I just noticed that I didn't explictly `return` here and I wanted to flag it in case you think it should be added for consistency

Reply:
Its fine! i personally prefer to have an explicit return, but Ruby automatically returns the last line of a function, and many people thinks that that's idiomatic Ruby.

I prefer not to do that, because not many people know Ruby to that degree of confidence, so being explicit makes the code more readable for everyone.